src/repl/repl_commands.cpp

```
@@ -1088,35 +1132,41 @@ bool dispatch_command(const std::string& command, const std::string& arg, ReplSt
+      // Ask LLM to summarize
+      std::vector<Message> sum_msgs = {{"system",
+                                        "Summarize this conversation in 2-4 bullet points. "
+                                        "Include: topics discussed, decisions made, and any pending items. "
+                                        "Be concise. Respond in the same language as the conversation."},
+                                       {"user", conversation}};
+      std::string summary;
+      s.stream_chat(sum_msgs, [&summary](const std::string& token) {
+        summary += token;
+        return true;
+      });
+      if (summary.empty()) {
```

Clang (14.0.6)

[note] +3, including nesting penalty of 2, nesting level increased to 3

(clang)